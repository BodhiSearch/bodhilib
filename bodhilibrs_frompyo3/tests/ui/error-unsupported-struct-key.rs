use frompyo3::FromPyO3;

#[derive(FromPyO3)]
#[frompyo3(unknown = "Resource")]
struct TestUnsupportedStructKey {}

fn main() {}
