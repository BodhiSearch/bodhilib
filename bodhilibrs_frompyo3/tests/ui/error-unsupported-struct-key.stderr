error: Unsupported struct attribute key: 'unknown'
 --> tests/ui/error-unsupported-struct-key.rs:4:12
  |
4 | #[frompyo3(unknown = "Resource")]
  |            ^^^^^^^
